src/data/data.ts

```diff
@@ -112,11 +124,12 @@ export const budgetOptions = [
   },
 ]
 
+// TODO(beshaya): Move RiskProfile to it's own file.
```

Comment:
Do you wanna do that in this PR or are you saving it for a future one?

Reply:
I'm intentionally not doing it in this PR because it would make the diff harder to read.